runtime/src/vm/run.rs

```diff
@@ -83,6 +83,7 @@ pub fn exec(fp: &str) {
   let mut rt = runtime::Builder::new()
     .basic_scheduler()
     .enable_time()
+    .enable_io()
```

Comment:
This one line contains multitudes.

Reply:
![multitudes](http://weneedfun.com/wp-content/uploads/2016/04/Walt-Whitman-Quotes-9.jpg)

Reply:
1. It contains multitudes of source code behind it.
2. It turns on IO in Tok.io, an IO scheduling library. What was it before? Just Tok? (It contradicts itself, too!)

Reply:
that multitude alluded me for longer than I really wish it had...